packages/fast-usdc/test/chain-info.test.js

```diff
@@ -0,0 +1,61 @@
+import test from 'ava';
+import fetchedChainInfo from '@agoric/orchestration/src/fetched-chain-info.js';
+
+// List of chains to test to ensure an IBC connection exists from noble to them
+// XXX: Some missing connections.
+const fastUsdcDestinationChains = [
```

Comment:
should this be available outside the test? e.g. for the contract to handle differently a destination that isn't in this list?

Reply:
Not sure. Possibly something to ensure we have a testing plan for... if we get a destination that's not supported.